### DISCUSSION TITLE
compiling on visual studio 2022, SDL.h No such file or diretory

### DISCUSSION THREAD

**Discussion by ghost** [stella-emu/stella#1041] [Q&A] [answered]
i ve open /src/os/windows/Stella.sln and
i'm trying to compiling on visual studio 2022 and i m getting  "SDL.h No such file or diretory" message
can you help ?

![image](https://github.com/user-attachments/assets/2686db34-b96c-4a4c-92f5-2b0eeed57b62)

**Comment by thrust26** [comment, marked as answer]
SDL2 is not part of the repository. Please check the documentation on our website how to compile Stella yourself.
